crates/egui/src/ui.rs

```diff
@@ -521,8 +521,7 @@ impl Ui {
     pub fn disable(&mut self) {
         self.enabled = false;
         if self.is_visible() {
-            self.painter
-                .set_fade_to_color(Some(self.visuals().fade_out_to_color()));
```

Comment:
We should also remove `Visuals::fade_out_to_color` too, and replace it with `fn fade_out_opacity(&self) -> f32 { 0.5 }` and use it for `Visuals::gray_out`